app/helpers/settings_helper.rb

```diff
@@ -7,6 +7,7 @@ module SettingsHelper
     es: 'Español',
     eo: 'Esperanto',
     fr: 'Français',
+    it: 'Italiano'
```

Comment:
Come on, one day I'll finally be able to submit something **totally** correct, I'm learning, never lose hope. 😂 ❤️ 
(Thank you for the correction)